nomic/data_operations.py

```diff
@@ -260,8 +333,8 @@ def get_topic_density(self, time_field: str, start: datetime, end: datetime):
             A list of `{topic, count}` dictionaries, sorted from largest count to smallest count.
         """
         data = AtlasMapData(self.projection, fields=[time_field])
-        time_data = data._tb.select([self.id_field, time_field])
-        merged_tb = self._tb.join(time_data, self.id_field, join_type="inner").combine_chunks()
+        time_data = data.tb.select([self.id_field, time_field])
+        merged_tb = self.tb.join(time_data, self.id_field, join_type="inner").combine_chunks()
```

Comment:
Why this change?

Reply:
We used to guarantee _tb was loaded when object was constructed because of the prefetch tiles logic. Now that it's removed, the download step only comes up if you call the tb property